WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -0,0 +1,136 @@
+import Foundation
+
+let AccountSettingsServiceChangeSaveFailedNotification = "AccountSettingsServiceChangeSaveFailed"
+
+struct AccountSettingsService {
+    let remote: AccountSettingsRemote
+    let accountID: Int
+
+    private let context = ContextManager.sharedInstance().mainContext
```

Comment:
The context should be an initialization parameter I'd think instead of tying it to a particular context.